lib/private/BackgroundJob/JobList.php

```diff
@@ -382,4 +386,23 @@ public function resetBackgroundJob(IJob $job): void {
 			->where($query->expr()->eq('id', $query->createNamedParameter($job->getId()), IQueryBuilder::PARAM_INT));
 		$query->executeStatement();
 	}
+
+	public function countByClass(): array {
+		$query = $this->connection->getQueryBuilder();
+		$query->select('class')
+			->selectAlias($query->func()->count('id'), 'count')
+			->from('jobs')
+			->orderBy('count')
+			->groupBy('class');
+
+		$result = $query->executeQuery();
+
+		$jobs = [];
+		while ($row = $result->fetch()) {
```

Comment:
```suggestion
		while (($row = $result->fetch()) !== false) {
```
I do not know if it is possible to end up with '0' string in class column but better safe than sorry, we have been bitten by falsy values in database columns before.